.cargo/config.toml

```diff
@@ -14,4 +14,8 @@ rustflags = [
     "-Ctarget-feature=-multivalue",
     "-Ctarget-feature=-atomics",
     "-Clink-arg=--export-table",
+    "-Copt-level=z",
```

Comment:
Have you benchmarked this? We started with `opt-level=z` and we got a massive speed-up when we switched to `opt-level=3`.